lib/autoproj/templates/build_report_helper.erb

```diff
@@ -0,0 +1,21 @@
+<% @first_line = true -%>
+{
+   "build_report" : {
+       "packages": [<% package_list.each do |pkg_name| -%>
+<% if @first_line == false  %>           },    <% end -%>
+<% @first_line = false  %>
+           {
+               "name": "<%= Autobuild::Package[pkg_name].name.to_s %>",
+               "time": "<%= Time.now.to_s %>",
+               "import_invoked": "<%= Autobuild::Package[pkg_name].import_invoked?.to_s %>",
```

Comment:
You don't need the `#to_s` in a `<%= ... %>` template, it's implicit (otherwise all of them would need it anyways ...).

Reply:
To be honest, I don't think a template is the right tool for this... Why not just generate a hash with the data and output it using `JSON#pretty_generate` ? it would be much easier to maintain and  we wouldn't have to worry about the json format in the template file...